docs/alerts.md

```diff
@@ -51,15 +129,91 @@ Elasticsearch will attempt to relocate shards away from a node for [reaching the
 
 ### Troubleshooting
 
-Check the disk space of the node.
-
+1. Identify the node on which elasticsearch is deployed:
+    ```
+   oc -n openshift-logging get po -o wide
+   ```
+
+2. Check the disk space on each node:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_nodes/stats/fs?pretty
+   ```
+
+3. Check if the cluster is rebalancing:
+   ```
+   oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/health?pretty | grep relocating_shards
+   ```
+
+4. If there are any relocating shards present then:
+    - The High Watermark which is 90% by default has been exceeded.
+    - The shards will start relocating to a different node which has low disk usage and hasn’t crossed any watermark threshold limits.
+    - To allocate shards to this particular node, free up some spaces:
+        - Increase disk space on all nodes.
+        - If increasing disk space isn’t possible, then add a new data node to the cluster.
+        - If adding a new data node is problematic, then reduce the number of replicas
+            - Preferably on older indices.
+            - Get the index settings:
+              ```
+              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty
+              ```
+            - Identify the number of replicas from the output of the above command.
+            - Lower the number of replicas if possible:
```

Comment:
this would need to be cluster-wide... the operator would try to adjust them afterwards I believe